samples/highcharts/blog/simple-Logarithmic-graph/demo.js

```diff
@@ -0,0 +1,56 @@
+Highcharts.chart("container", {
+  title: {
+    text: null
+  },
+
+  xAxis: {
+    categories: [
+      "Monday",
+      "Tuesday",
+      "Wednesday",
+      "Thursday",
+      "Friday",
+      "Saturday",
+      "Sunday"
+    ]
+  },
+
+  yAxis: {
+    title: {
+      text: "Price"
+    },
+    type: "logarithmic",
+    labels: {
+      format: "{text}$"
```

Comment:
I _think_ the [dollar sign should be before the number](https://english.stackexchange.com/questions/11326/what-is-the-difference-between-20-and-20) 🤔  . If so, that should be fixed throughout all these samples.